### PR TITLE
Get subscription keys and endpoints from env vars

### PR DESCRIPTION
**Pull request by v-jasowells**

### CODE REVIEW

File: Detect.py [hunk 1]
```diff
@@ -7,25 +7,22 @@
...
-    subscriptionKey = os.environ['TRANSLATOR_TEXT_KEY']
-else:
-    print('Environment variable for TRANSLATOR_TEXT_KEY is not set.')
-    exit()
-# If you want to set your subscription key as a string, uncomment the next line.
-#subscriptionKey = 'put_your_key_here'
+key_var_name = 'TRANSLATOR_TEXT_SUBSCRIPTION_KEY'
+if not key_var_name in os.environ:
+    raise Exception('Please set/export the environment variable: {}'.format(key_var_name))
+subscriptionKey = os.environ[key_var_name]
```
Comment (erhopf) [inline Detect.py]: Hey Jason, can we adjust this to `subscription_key` to match python standards. This was an error in my old samples as well. This is true for all samples.

**Review by erhopf — COMMENTED** [review]
One minor edit.

File: Detect.py (hunk 1 above)
Comment (v-jasowells) [reply Detect.py]: Done.

**Review by erhopf — APPROVED** [review]
@Jann-Skotdal - Can you merge these changes please ;)

**Pull request merged**